### ISSUE TITLE
[Suggestion] Add math support

### ISSUE THREAD

**Issue by C0ffeeCode** [trimental/inlyne#56]
Hello there ;)

I would like to use Inlyne for viewing my documents which contain (a lot) of mathematical notations
and think support for math grammar would be a great addition to inlyne.

**Comment by matwatson** [comment]
Mathjax support possibly?
Here's how Mathjax (https://www.mathjax.org/#gettingstarted) suggests adding math to HTML.

`<script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
<script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>`

**Comment by CosmicHorrorDev** [comment]
I don't think that would be a viable approach for us unfortunately since it would force us to allow for executing JavaScript which is a decent chunk of browers' complexity

**Comment by LoganWalls** [comment]
Perhaps [katex-rs](https://github.com/xu-cheng/katex-rs) or [ReX](https://github.com/KenyC/ReX) might viable alternatives?
I would also love to see math support (but I understand that it's a fairly complex feature).
